admiral.sh

```diff
@@ -32,6 +32,7 @@ readonly AWSCLI_VERSION=1.10.63
 readonly API_TIMEOUT=600
 readonly ADMIRAL_PORT=50003
 export LC_ALL=C
+export docker_restart=false
```

Comment:
When we change docker options, to add shippable docker options. We need to restart docker after that, so this acts as a flag.